server/http.go

```diff
@@ -940,7 +940,7 @@ func (p *Plugin) slackAttachmentToUpdatePMI(currentValue, channelID string) *mod
 func (p *Plugin) checkChannelPreference(channelID string) (bool, int, error) {
 	channel, appErr := p.API.GetChannel(channelID)
 	if appErr != nil {
-		return false, http.StatusInternalServerError, appErr
+		return false, http.StatusInternalServerError, errors.New(appErr.Message)
```

Comment:
was this change necessary?

Reply:
Yes, `appErr` is of a different type, and the return type is different. Also, to get a proper error message, we are just returning the `Message` field from the `appErr`.